cpp/tests/transform/row_bit_count_test.cu

```diff
@@ -47,7 +47,7 @@ TYPED_TEST(RowBitCountTyped, SimpleTypes)
   // expect size of the type per row
   auto expected = make_fixed_width_column(data_type{type_id::INT32}, 16);
   cudf::mutable_column_view mcv(*expected);
-  thrust::fill(rmm::exec_policy(0),
+  thrust::fill(thrust::device,
```

Comment:
These should still use the rmm exec policy so they use the pool that's initialized for all tests.